src/main/java/uk/gov/ons/ctp/response/casesvc/representation/CaseDTO.java

```diff
@@ -20,8 +20,8 @@
    * enum for Case event
    */
   public enum CaseEvent {
-    ACCOUNT_CREATED, ACTIVATED, DEACTIVATED, DISABLED, HOUSEHOLD_PAPER_REQUESTED, INDIVIDUAL_RESPONSE_REQUESTED,
-    REPLACED, CASE_CREATED
+    ACCOUNT_CREATED, ACTIVATED, DEACTIVATED, DISABLED, DISABLE_RESPONDENT_ENROLMENT,
```

Comment:
Should there be a new event? I feel like the event is still DISABLED just a new category